codespell_lib/data/dictionary.txt

```diff
@@ -1244,18 +1244,17 @@ analitic->analytic
 analitical->analytical
 analitically->analytically
 analiticaly->analytically
-analize->analyze
-analized->analyzed
-analizer->analyzer
-analizes->analyzes
-analizing->analyzing
+analize->analyze, analyse,
+analized->analyzed, analysed,
+analizer->analyzer, analyser,
+analizes->analyzes, analyses,
+analizing->analyzing, analysing,
```

Comment:
If it's misspelled with the letter of said dialect then we only correct it for said dialect.

Reply:
Oh yeah, sorry. That's obvious. I'll change it now.